src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -609,10 +609,10 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
       scheduleTransition(WorkflowFailed)
       stay()
     case Event(message @ AsyncFailure(t), _) =>
-      // This is the unusual combination of debug + throwable logging since the expectation is that this will eventually
+      // This is the unusual combination of warn + throwable logging since the expectation is that this will eventually
       // be logged in the case above as an error, but if for some weird reason this actor never ends up in that
```

Comment:
I'm happy to do that once we find out what's really going on, but I'm current not sure what I'd write / unwrite.  :frowning: